tests/core.py

```diff
@@ -42,7 +42,7 @@
 DEFAULT_DATE_ISO = DEFAULT_DATE.isoformat()
 DEFAULT_DATE_DS = DEFAULT_DATE_ISO[:10]
 TEST_DAG_ID = 'unit_tests'
-configuration.test_mode()
+
```

Comment:
@bolkedebruin I deleted this one because it was already called on line 22 of `tests/core.py` -- but to be completely honest, I'm not sure if `configuration.test_mode()` actually does anything?

This is the entire function:
```python

def test_mode():
    conf = ConfigParserWithDefaults(defaults)
    conf.read(TEST_CONFIG)
```

It creates a new `conf` object with the test parameters, but doesn't return it or overwrite the existing one. And it doesn't change values that were already read-in at the module level (like `DAGS_FOLDER` at the top of `models.py`)